tests/sample_data/premiere_generators.xml

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it's a shame the BOM doesn't show in the diff

Reply:
yeah!  oddly enough `git diff` from the command line did show it to me